src/components/views/employees/EmployeeCard.tsx

```diff
@@ -1,50 +1,49 @@
 import Avatar from '@mui/material/Avatar';
 import ButtonBase from '@mui/material/ButtonBase';
 import Stack from '@mui/material/Stack';
-import { Theme } from '@mui/material/styles';
 import Typography from '@mui/material/Typography';
-import { makeStyles } from '@mui/styles';
 import Link from 'next/link';
 import { getEmoji } from 'utils/utils';
 
-const useStyles = makeStyles((theme: Theme) => ({
-  root: {
-    borderRadius: theme.spacing(2),
-    backgroundColor: theme.palette.background.paper,
-    padding: theme.spacing(1),
-    display: 'grid',
-    gridTemplateColumns: '2fr 3fr',
-    width: '100%',
-  },
-}));
-
 export type EmployeeCardProps = {
   id: number;
   firstName: string;
   lastName: string;
   imageUrl: string;
   role: string;
   gender: string;
+  nrOfMyTasks: number;
 };
 
-const EmployeeCard = ({ id, firstName, lastName, imageUrl, role, gender }: EmployeeCardProps) => {
-  const classes = useStyles();
+const EmployeeCard = ({ id, firstName, lastName, imageUrl, role, gender, nrOfMyTasks }: EmployeeCardProps) => {
   const emoji = getEmoji(role, gender);
 
   return (
     <Link href={`/ansatt/${id}`} passHref>
-      <ButtonBase className={classes.root}>
+      <ButtonBase
+        sx={{
+          borderRadius: 2,
+          backgroundColor: 'background.paper',
+          padding: 1,
+          display: 'grid',
+          gridTemplateColumns: '1fr 2fr',
+          width: '100%',
+          alignItems: 'start',
+        }}
+      >
         <Avatar alt={`${firstName[0]}.${lastName[0]}`} src={imageUrl} sx={{ width: 80, height: 80 }} />
-        <Stack spacing={1} sx={{ justifyContent: 'center' }}>
+        <Stack padding={1} spacing={0}>
           <Typography variant='h4'>
             {firstName} {lastName[0]}.
           </Typography>
-          <Typography variant='h5'>
+          <Typography variant='caption'>
             {emoji}
 
             {role || 'Ukjent'}
           </Typography>
+          {Boolean(nrOfMyTasks) && <Typography variant='caption'>{`Du har ${nrOfMyTasks} oppgave${nrOfMyTasks > 1 ? 'r' : ''}`}</Typography>}
```

Comment:
Foreslår at det står noe annet enn 'Du har 1 oppgave' da det er lite presist. 1 oppgave hva da?

Legger ved et forslag

```suggestion
          {Boolean(nrOfMyTasks) && <Typography variant='caption'>{`${nrOfMyTasks} oppgave${nrOfMyTasks > 1 ? 'r' : ''} gjenstår`}</Typography>}


```